Alignment/TrackerAlignment/test/mcMisalignmentScaler_cfg.py

```diff
@@ -25,6 +25,7 @@
     )
 )
 process.mcMisalignmentScaler.pullBadModulesToIdeal = False
+process.mcMisalignmentScaler.outlierPullToIdealCut = 0.1
```

Comment:
@ghellwig can you please add a comment here explaining the choice of this threshold?

Reply:
ping @ghellwig

Reply:
@arunhep sorry, your question slipped through the cracks.
This cfg file is just an arbitrary example and as you can see above the default given in `fillDescriptions` is a negative value, i.e. outlier modules are *not* pulled to ideal.